File: Composite/python/TaskManagementApp.py
from Tasks import Task, SimpleTask, TaskList

def main() -> None:
    simple_task_1: Task = SimpleTask(title="Complete Coding")
    simple_task_2: Task = SimpleTask(title="Write Documentation")
    
    project_tasks: Task = TaskList(title="Project Tasks")
    project_tasks.add_task(simple_task_1)
    project_tasks.add_task(simple_task_2)
    
    phase_1_tasks: Task = TaskList(title="Phase 1 Tasks")
    phase_1_tasks.add_task(SimpleTask(title="Design"))
    phase_1_tasks.add_task(SimpleTask(title="Development"))
    
    project_tasks.add_task(phase_1_tasks)
    
    project_tasks.display()
    

if __name__ == '__main__':
    main()

File: Composite/ts/TaskManagementApp.ts
import { Task, SimpleTask, TaskList } from "./Task";


const simpleTask1: Task = new SimpleTask("Complete Coding");
const simpleTask2: Task = new SimpleTask("Write Documentation");

const projectTasks: TaskList = new TaskList("Project Tasks");
projectTasks.addTask(simpleTask1);
projectTasks.addTask(simpleTask2);

const phase1Tasks: TaskList = new TaskList("Phase 1 Tasks");
phase1Tasks.addTask(new SimpleTask("Design Application"));
phase1Tasks.addTask(new SimpleTask("Develop Application"));

projectTasks.addTask(phase1Tasks);

projectTasks.display();
